zipkin/src/main/java/zipkin/Annotation.java

```diff
@@ -101,11 +100,6 @@ public Annotation build() {
   }
 
   @Override
-  public String toString() {
-    return JsonCodec.ANNOTATION_ADAPTER.toJson(this);
```

Comment:
removed toStrings on leave nodes while leaving json toString on entrypoint ones (Span and DependencyLink). This reduces the amount of internal class leak